### DISCUSSION TITLE
Validation demande de réservation

### DISCUSSION THREAD

**Discussion by Cyrilb42** [JeromeDevome/GRR#556] [Q&A]
Bonjour,

Nous rencontrons une problématique avec la validation des demandes de réservations (soumis à modération).

Il arrive que certains de nos utilisateurs (pas gestionnaire de la ressource en question) effectuent des demandes de réservations et qu'ils les valident eux-mêmes.
Ils ont cette possibilité parce que dans GRR, ils sont désignés comme gestionnaire (d'une autre ressource).

Une amélioration est elle possible afin que seule les gestionnaires de cette ressource soient en mesure de valider.

Le champ "Type" contenant le libellé "Validée" (qui valide donc la demande de résa) est proposé aux gestionnaires et administrateurs dans les réglages de l'outil.
Peut-on ajouter une option "gestionnaire de la ressource" ?

Merci

Infos GRR
Version de GRR : 4.3.2
Version PHP : 8.3

**Comment by JeromeDevome** [comment]
Bonjour,
Pourquoi ne pas utiliser les réservations modérés ?

**Comment by Cyrilb42** [comment]
Bonjour,

Nous utilisons les réservations modérés en complément du champ Type pour déclencher l'envoi de mail, mais les réservations restent en attente de modération (fonctionnement maison en place depuis plusieurs années ...).

Je reformule la demande, est-il possible lors de la déclaration d'un utilisateur comme gestionnaire d'une ressource X qu'il n'ait accès uniquement qu'à cette ressource en gestion.
Cet utilisateur gestionnaire d'une ressource X créé une demande de réservation sur une ressource Y il devrait être considéré comme un simple utilisateur (mais parce qu'il est gestionnaire, il a des options supplémentaires qui lui permettent de valider la demande).

Merci à vous
Bonne journée
